lib/omnibus/health_check.rb

```diff
@@ -35,6 +35,7 @@ class HealthCheck
       /libutil\.so/,
       /linux-vdso.+/,
       /linux-gate\.so/,
+      /libffi\.so/,
```

Comment:
This points to a linking bug in the underlying software definitions. Something isn't linking against the libffi we build but instead linking against the system libffi. The health check if failing just as it should in this situation.